build/ToolsetPackages/InternalToolset.csproj

```diff
@@ -3,7 +3,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net46</TargetFramework>
-    <NonShipping>true</NonShipping>
```

Comment:
What does NonShipping actually mean these days? Because this 'project' definitely _isn't_ shipping...

Reply:
Doesn't mean anything if we don't import the targets that use it, which is what this change is doing.